documentation/contributing/introduction.adoc

```diff
@@ -1,21 +1,71 @@
 = Introduction
 
-This guide is meant to assist contributors to the Strimzi documentation.
+This guide provides information to assist contributors to the Strimzi documentation.
 
-== Dependencies
+Strimzi documentation is written in Asciidoc.
 
-* asciidoctor
-* python
+== Documentation tools
+
+https://asciidoctor.org/[`asciidoctor`^]:: Documentation generation tool
+https://www.gnu.org/software/make/[`make`^]:: Make build system to build complete documentation including API reference
+https://github.com/mikefarah/yq[`yq`^]:: YAML build tool to build documentation using _make_ targets
+
+For most documentation updates, `asciidoctor` offers the simplest way to check the build.
 
 == Project layout
 
+Strimzi uses git to manage repositories.
+
+https://github.com/strimzi/strimzi-kafka-operator[`strimzi-kafka-operator`^] (GitHub):: The public GitHub repo hosts all of the Strimzi code and documentation.
+https://github.com/strimzi/strimzi-kafka-bridge[`strimzi-kafka-bridge`^] (GitHub):: The public GitHub repo hosts all of the Strimzi Kafka Bridge code.
```

Comment:
We have the bridge here, should we add the `oauth` repo as well?

Reply:
@PaulRMellor Can you have a look at this? It would be great if we could close this PR finally.